orttraining/orttraining/test/optimizer/graph_transform_test.cc

```diff
@@ -1302,6 +1303,191 @@ TEST_F(GraphTransformationTests, MegatronBARTSelfAttentionPartitionCorrectnessTe
 // end of USE_CUDA
 #endif
 
+/*
+Test graph as below.
```

Comment:
Need another test to cover case we discussed before:
      graph input [1, 1, 256, 256] (float)  scalar_0     graph input [1, 1, 256, 256] (float)
                                         \   /          /
                                           Div         Div -- scalar_1
                                             \         /
                                               Add
             Other type op                     /
                        \                     /
                          \                /
                                Add
                                 |
                               Identity
                                 |
                graph out [1, 1, 256, 256] (float)